docs/spec_draft.md

```diff
@@ -2126,6 +2128,87 @@ More formally, for each element `x`: `real(x) = is_complex(x) ? x.real : x`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.recv
+
+### Semantics
+
+Receives data to a partition from a [stablehlo.send](#stablehlosend)
+instruction, in another partition, that shares the same channel handle
+`channel_handle`.
+
+`stablehlo.recv`, paired with the corresponding [stablehlo.send](#stablehlosend)
```

Comment:
we can refactor this into a separate section
Point to Point ops
or
Process to Process Ops

as opposed to Collective Ops.

Reply:
Nice one! Incorporated.